x-pack/legacy/plugins/ml/public/data_frame/common/job.ts

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { PivotAggDict } from './pivot_aggs';
+import { PivotGroupByDict } from './pivot_group_by';
+
+export type IndexName = string;
+export type IndexPattern = string;
+export type JobId = string;
+
+export interface DataFrameJob {
```

Comment:
Good point! I'll change this in a follow up too though, because `DataFrameJob` is already used in other parts of the UI e.g. the list page and I'd like to avoid touching much more unrelated code to the advanced editor. It just shows up here as a new line because I moved it from another file to a more general place.